api_docs/changelog.md

```diff
@@ -20,6 +20,12 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 10.0
 
+**Feature level 326**
```

Comment:
@sahil839 I noticed that `version.py/API_FEATURE_LEVEL` wasn’t bumped to 326 for this change.

Reply:
Thanks for noticing. Opened #32573.